library/model/book.py

```diff
@@ -7,23 +7,11 @@
 
 from library.persistence.storage import LibraryRepository
 
-
 class Book:
-    title: str
-    authors: list[Author]
-    publisher: Publisher
-    publication_date: datetime
-    genres: list[Genre]
-    pages: int
-    isbn: str
-
-    existing_items: int
-    borrowed_items: int
-
-    _book_type: str
-    duration: int = 0
-
-    def __init__(self, title, authors, publisher, pub_date, genres, pages, isbn, type, duration=0, existing_items=1, borrowed_items=0):
+    due_date: datetime
+    weekly_fee: float
```

Comment:
Variables are unused.

Reply:
However, you could consider separation of concerns. The book class represents a collection of books, while a `due_data` variable is associated with a unique copy of a book. Similarly logic applies to the `weekly_fee`.